huaweicloud/services/vpc/data_source_huaweicloud_networking_port_v2.go

```diff
@@ -102,6 +92,24 @@ func DataSourceNetworkingPortV2() *schema.Resource {
 				Computed: true,
 				Elem:     &schema.Schema{Type: schema.TypeString},
 			},
+
+			// marked as deprecated
+			"admin_state_up": {
+				Type:       schema.TypeBool,
+				Optional:   true,
+				Computed:   true,
+				Deprecated: "this field is deprecated",
```

Comment:
```suggestion
				Description:  "schema: Deprecated; this field is deprecated",
```

Reply:
it's safe to deprecate them, we don't use them for a long time